examples/specs/wheat_wages.vl.json

```diff
@@ -74,7 +74,7 @@
         { "calculate": "((!datum.commonwealth && datum.index % 2) ? -1: 1) * 2 + 95", "as": "offset" },
         { "calculate": "95", "as": "y" }
       ],
-      "mark": {"type": "bar", "stroke": "#000"},
+      "mark": {"type": "bar", "stroke": "#000", "orient": "vertical"},
```

Comment:
This one has all of y, y2, x, x2, I think it should just be a rect.

Reply:
It works!